CMakeLists.txt

```diff
@@ -877,7 +885,7 @@ set (CMAKE_CURRENT_BINARY_DIR ${save_CBD})
 
 add_library (pbufs STATIC ${PROTO_SRCS} ${PROTO_HDRS})
 if ((NOT TARGET protobuf::libprotobuf) AND (NOT Protobuf_LIBRARIES))
-  message ("protobuf lib not found - adding unity source for lib")
+  message (STATUS "protobuf lib not found - adding unity source for lib")
```

Comment:
Would it be possible to remove the whole vendored protobuf code from this repository and have it as an external dependency instead? The vendored version is ancient...

Reply:
@MarkusTeufelberger I agree..and it's only used for windows which is confusing. Now that I've refactored the build, I'm going to look at the NIH stuff more closely